src/test/resources/application-tracing-test.properties

```diff
@@ -1,4 +1 @@
-terra.common.tracing.samplingRate: 1.0
-# Empty list of urlPatterns to allow all URLs to be traced.
-terra.common.tracing.urlPatterns:
```

Comment:
Since urlPatterns is no longer a setting, does this mean all URLs are traced?

Reply:
all urls are traced